tests/examples/wibox/widget/checkbox/custom.lua

```diff
@@ -0,0 +1,35 @@
+local parent    = ... --DOC_NO_USAGE --DOC_HIDE
+local wibox     = require( "wibox"     ) --DOC_HIDE
+local beautiful = require( "beautiful" ) --DOC_HIDE
+local gears     = {shape = require("gears.shape")} --DOC_HIDE
+
+parent:add( --DOC_HIDE
+    wibox.widget {
+        checked       = true,
+        color         = beautiful.bg_normal,
+        paddings      = 2,
+        forced_width  = 20, --DOC_HIDE
+        forced_height = 20, --DOC_HIDE
+        check_shape   = function(cr, width, height)
+            local rs = math.min(width, height)
+            
+            cr:move_to( 0  , 0  )
+            cr:line_to( rs , 0  )
+            cr:move_to( 0  , 0  )
+            cr:line_to( 0  , rs )
+            cr:move_to( 0  , rs )
+            cr:line_to( rs , rs )
+            cr:move_to( rs , 0  )
+            cr:line_to( rs , rs )
+            cr:move_to( 0  , 0  )
+            cr:line_to( rs , rs )
+            cr:move_to( 0  , rs )
+            cr:line_to( rs , 0  )
+            cr:close_path()
```

Comment:
Why this `close_path`? I'd leave it out. It doesn't really do much here and seems weird to me. The other subpaths aren't explicitly closed either.